plugins/nginx-vhosts/functions

```diff
@@ -142,19 +182,29 @@ nginx_build_config() {
         local DOKKU_APP_PORT_FILE="${DOKKU_APP_IP_FILE//IP/PORT}"
         local DOKKU_APP_LISTENER_IP=$(< "$DOKKU_APP_IP_FILE")
         local DOKKU_APP_LISTENER_PORT=$(< "$DOKKU_APP_PORT_FILE")
-
-        if [[ -z "$RAW_TCP_PORTS" ]]; then
```

Comment:
This is unnecessary as we build `DOKKU_PROXY_PORT_MAP` from `RAW_TCP_PORTS` if it exists.